scripts/popup.js

```diff
@@ -498,6 +522,8 @@ $('#first_capture').click(first_capture)
 $('#fb_share').click(social_share)
 $('#twit_share').click(social_share)
 $('#linkedin_share').click(social_share)
+$('#copy-link').click(social_share)
+$('#email-share').click(social_share)
```

Comment:
Add the href attribute directly inside `window.onload` function !

Reply:
Thanks.

Reply:
I made the required changes. Is there a better way to do this?